src/main/java/com/google/api/generator/engine/writer/JavaWriterVisitor.java

```diff
@@ -124,6 +125,16 @@ public void visit(AnnotationNode annotation) {
     newline();
   }
 
+  @Override
+  public void visit(NewObjectValue newObjectValue){
+    buffer.append(newObjectValue.value()).append(LEFT_PAREN);
```

Comment:
This should be part of NewObjectValue, since it would not be complete without the arguments.

Reply:
Resolving in favor of the `NewObjectExpr` comments.